fighters/robot/src/acmd/specials.rs

```diff
@@ -310,7 +336,11 @@ unsafe extern "C" fn effect_specialhi(agent: &mut L2CAgentBase) {
             }
             LAST_EFFECT_SET_RATE(agent, 0.75);
         }
-        wait(lua_state, 15.0);
+        wait(lua_state, 10.0);
+    }
+    frame(lua_state, 27.0);
+    if is_excute(agent) {
+        EFFECT_OFF_KIND(agent, Hash40::new("robot_lamp_l"), false, false);
```

Comment:
Right above this you loop 20 times waiting 10 frames each. That means the animation is at frame 215 here, not 27 as you expect. If the animation is less than 215 frames this code will never be reached